crates/bevy_color/Cargo.toml

```diff
@@ -18,6 +18,7 @@ serde = "1.0"
 thiserror = "1.0"
 wgpu = { version = "0.19.1", default-features = false }
 encase = { version = "0.7", default-features = false }
+aquamarine = "0.5"
```

Comment:
Does this need to be an ordinary dependency? Can't it be a dev-dep, since it's only used when generating docs?

Reply:
So I _thought_ it could be a dev dependency, but that doesn't appear to work. As an alternative (since I don't expect this diagram to change often), we could just render the diagram to an SVG and include it directly, removing the `aquamarine` dependency entirely.

Reply:
Let's go with that :) That will also make it easier to include in e.g. the release notes or in the book.

Reply:
Ok I've implemented that, and left a small README next to the diagram `.mmd` file explaining how to generate the SVG. Annoyingly, the SVG needs to be wrapped in `<div>` tags to suppress `rustdoc` warning about CSS links inside the SVG, but it's a minor problem.